ParisBNF/abb/BNFabb202.xml

```diff
@@ -0,0 +1,483 @@
+                          <decoNote type="ornamentation" xml:id="d1">
+                          <locus target="#1r"/>
+                          <desc>Monochrome (red) ornamental band.</desc>
+                        </decoNote>
+                        </decoDesc>
+
+                        <additions>
+                          <list>
+                            <item xml:id="a1">
+                              <locus target="#1ra"/>
+                               <desc type="Unclear">A short prayer defying Satan.</desc>
+                                 <q xml:lang="gez">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩አምላክ። በቅደስት፡ ሥላሴ፡
+                                 እንዘ፡ ኣምን፡ ወእት፡ መኃፀን፡ እከህደከ፡ ሰይጣን፡ በቅድመ፡ ዛቲ፡ እምየ፡ ቅድስት፡ ቤተ፡ ክርስቲያን፡ እንተ፡
+                               ይእቲ፡ ስማዕየ፡ ማርያም። ጽዮን፡ ለዓለመ፡ ዓለም።</q>
+                                   <note>Written in a good hand.</note>
+                             </item>
+
+                             <item xml:id="a2">
+                               <locus from="1ra" to="1rb"/>
+                               <desc type="GuestText"><title type="complete" ref="LIT5851Prayer"/></desc>
+                               <q xml:lang="gez">ነአኵተከ፡ እግዚኦ፡ ንሴብሐከ። ንባርከከ፡ እግዚኦ፡ ወንትአመነከ፡ ንገኒ፡ ለከ፡
+                                  እግዚኦ፡ ወንትቀነይ፡ ለስመከ፡ ቀዱስ፡</q>
+                               <note>Separated from <ref target="#a1"/> by a dotted line.</note>
+                              </item>
+
+                              <item xml:id="a3">
+                                <locus from="1rb" to="1va"/>
+                                <desc type="GuestText"><title type="complete" ref="LIT2168PaterN"/></desc>
+                                <q xml:lang="gez">አቡነ᎓ ዘበሰማያት። ይትቀደስ፡ ሰምከ፡ ትምፃእ፡ መንገሥትከ፡ ወይኩን፡ ፈቃደከ፡</q>
+                                <note>Separated from <ref target="#a2"/> by a dotted line.</note>
+                                 </item>
+
+                                <item xml:id="a4">
+                                  <locus from="1va" to="1vb"/>
+                                  <desc type="GuestText"><title type="complete" ref="LIT6046InvocationMary"/></desc>
+                                  <q xml:lang="gez">በሰላመ᎓ ገብርኤል፡ መልአክ። ኦእግዝእትየ፡ ማርያም፡ ሰላም፡ ለኪ፡ ድንግልት፡ በኅሊናኪ፡ </q>
+                                   <note>Separated from <ref target="#a3"/> by a dotted line.</note>
+                                  </item>
+
+                                  <item xml:id="a5">
+                                    <locus target="#2r"/>
+                                     <desc type="PoemQene"/>
+                                       <q xml:lang="gez"></q>
+                                       <q xml:lang="en">For which generation among the number of generations,
+                                       </q>
+                                       <note>Qǝne-poem of the type <foreign xml:lang="gez">mawaddǝs</foreign>,
+                                     of Ṭǝbaba Krǝstos.</note>
```

Comment:
Is that the name of the author? Then it would be better to create an ID.

Reply:
Yes.